html-forms/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form</h1>
+
+<form action="" method="get">
+
+<!-- Email Input -->
+     <label for="email">Email: </label>
+      <input type="email" name="email" id="email"/>
+
+<br><br><!-- Drop Down List -->
+  <div>
+    <label for="options">Why is coding cool? </label>
+  </div>
+    <select id ="options" name="options">
+     <option value="Select an option">Select an option</option>
+     <option value="Beacuse it's interesting">Because it's interesting</option>
+     <option value="Beacuse it's fun"> Because it's fun</option>
+     <option value="Because it's powerful">Becasue it's powerful</option>
+    </select>
+
+
+<br><br><!-- Radio Buttons -->
+<div>
+  <label for="Favorite">Favorite Language (choose one):</label>
+  <input type="radio" name="favorite" value="HTML"> HTML
```

Comment:
The text following your radio inputs here should be the `label` for them. Make sure to include a proper `for` attribute to correlate to the `id` on the target input!